src/app/qgisapp.cpp

```diff
@@ -11408,6 +11415,15 @@ void QgisApp::keyPressEvent( QKeyEvent * e )
   }
 }
 
+void QgisApp::onTaskCompleteShowNotify( long taskId, int status )
+{
+  if ( status == QgsTask::Complete && !this->hasFocus() )
+  {
+    QString description = QgsApplication::taskManager()->task( taskId )->description();
```

Comment:
Better be safe here and make sure that QgsApplication::taskManager()->task( taskId ) returns something. It's possible it could return null in some circumstances.